template.yaml

```diff
@@ -71,8 +71,8 @@ Resources:
     Properties:
       BucketName:  !If
         - WithSuffix
-        - !Sub 'eventbridge-event-bodies-${AWS::AccountId}-${Suffix}'
-        - !Sub 'eventbridge-event-bodies-${AWS::AccountId}'
+        - !Sub 'eventbridge-event-bodies-${AWS::StackId}-${Suffix}'
```

Comment:
events bucket is very hard to delete when it is full. The expiration of the objects will make it easier to delete old empty buckets eventually.